rtc/jwt-authentication.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: default
+title: RTC JWT Authentication
+title_nav: JWT Authentication
+description: Guide on how to setup JWT Authentication for RTC
+keywords: jwt authentication
+---
+
+{% assign pluginname = "RTC" %}
+{% assign plugincode = "rtc" %}
+## Introduction
+
+RTC requires you to setup JSON Web Token (JWT) authentication. This is to ensure that only your authenticated users will be able to access and collaborate on documents.
+
+JWT is a standard authorization solution for web services and is documented in more detail at the [https://jwt.io/](https://jwt.io/) website. The guide aims to show how to setup JWT authentication for RTC.
+
+{% include auth/private-public-key-pairs-for-tiny-cloud-services.md %}
+
+{% include auth/jwt-provider-endpoint-url.md %}
+
+## JWT requirements
+
+{% include auth/jwt-supported-algorithms.md %}
+
+### Claims
+
+These are like options/data you can send with the JWT token.
```

Comment:
This info is now a table for RTC and a DT/DD for Drive - do we care that it's dissimilar?

Reply:
table doesn't really work for Drive because of the URL, hence the definition list.